res/values-es/strings.xml

```diff
@@ -5,27 +5,27 @@
     <string name="Apply">Aplicar</string>
     <string name="BangButtonExplanation">Buscar en otros lugares es fácil con !comandos</string>
     <string name="Cancel">Cancelar</string>
-    <string name="ChangingFontSize">Cambiando Tamaño de Letra&#8230;</string>
+    <string name="ChangingFontSize">Cambiando tamaño de letra&#8230;</string>
     <string name="Confirm">Confirmar</string>
-    <string name="ConfirmClearCacheAndCookies">El caché del navegador va a ser eliminado. ¿Estás seguro?</string>
-    <string name="ConfirmClearCookies">Todas las cookies del navegador serán eliminadas. ¿Estás seguro?</string>
-    <string name="ConfirmClearHistory"> Todo el historial de búsqueda será eliminado. ¿Estás seguro? </string>
-    <string name="ConfirmExternalBrowser">Por favor Confirma</string>
-    <string name="Continue">Continua</string>
+    <string name="ConfirmClearCacheAndCookies">Se eliminará el caché del navegador. ¿Estás seguro?</string>
+    <string name="ConfirmClearCookies">Se eliminarán todas las cookies del navegador. ¿Estás seguro?</string>
+    <string name="ConfirmClearHistory">Se eliminará todo el historial de búsqueda. ¿Estás seguro?</string>
+    <string name="ConfirmExternalBrowser">Por favor, confirma</string>
+    <string name="Continue">Continuar</string>
     <string name="Default">Predeterminado</string>
-    <string name="Defaults"> Restaurar ha Predeterminado </string>
-    <string name="DefaultsSetToast">Fuesntes restauradas a predeterminado</string>
+    <string name="Defaults">Restaurar ha predeterminado</string>
```

Comment:
This "ha" is without "h": "Restaurar **a** predeterminado". Although, I'd probably say: "Restaurar valores predeterminados" (Restore default values).